Cubical/Categories/Pullback.agda

```diff
@@ -13,32 +13,32 @@ private
   variable
     ℓ ℓ' : Level
 
-record Cospan (𝒞 : Precategory ℓ ℓ') : Type (ℓ-max ℓ ℓ') where
+record Cospan (C : Precategory ℓ ℓ') : Type (ℓ-max ℓ ℓ') where
   constructor cospan
   field
-    S₁ S₂ vertex : Precategory.ob 𝒞
-    s₁ : hom 𝒞 S₁ vertex
-    s₂ : hom 𝒞 S₂ vertex
+    S₁ S₂ vertex : Precategory.ob C
+    s₁ : C [ S₁ , vertex ]
+    s₂ : C [ S₂ , vertex ]
 
-record Cone {𝒞 : Precategory ℓ ℓ'} (cspn : Cospan 𝒞) (c : ob 𝒞) : Type (ℓ-max ℓ ℓ') where
+record Cone {C : Precategory ℓ ℓ'} (cspn : Cospan C) (c : ob C) : Type (ℓ-max ℓ ℓ') where
   constructor cone
   field
-    p₁ : hom 𝒞 c (Cospan.S₁ cspn)
-    p₂ : hom 𝒞 c (Cospan.S₂ cspn)
-    sq : seq 𝒞 p₁ (Cospan.s₁ cspn) ≡ seq 𝒞 p₂ (Cospan.s₂ cspn)
+    p₁ : C [ c , (Cospan.S₁ cspn)]
+    p₂ : C [ c , (Cospan.S₂ cspn)]
+    sq : p₁ ⋆⟨ C ⟩ (Cospan.s₁ cspn) ≡ p₂ ⋆⟨ C ⟩ (Cospan.s₂ cspn)
 
-record Pullback {𝒞 : Precategory ℓ ℓ'} (cspn : Cospan 𝒞) : Type (ℓ-max ℓ ℓ') where
+record Pullback {C : Precategory ℓ ℓ'} (cspn : Cospan C) : Type (ℓ-max ℓ ℓ') where
   constructor pullback
   field
-    c : ob 𝒞
+    c : ob C
     cn : Cone cspn c
-    universal : {c' : ob 𝒞} (cn' : Cone cspn c') → ∃![ f ∈ 𝒞 .hom c' c ] Σ[ q ∈ Cone.p₁ cn' ≡ 𝒞 .seq f (Cone.p₁ cn) ] (Cone.p₂ cn' ≡ 𝒞 .seq f (Cone.p₂ cn))
+    universal : {c' : ob C} (cn' : Cone cspn c') → ∃![ f ∈ C [ c' , c ] ] Σ[ q ∈ Cone.p₁ cn' ≡ f ⋆⟨ C ⟩ (Cone.p₁ cn) ] (Cone.p₂ cn' ≡ f ⋆⟨ C ⟩ (Cone.p₂ cn))
 
 -- whisker the parallel morphisms g and g' with f
-lPrecatWhisker : {𝒞 : Precategory ℓ ℓ'} {x y z : 𝒞 .ob} (f : 𝒞 .hom x y) (g g' : 𝒞 .hom y z) (p : g ≡ g') → 𝒞 .seq f g ≡ 𝒞 .seq f g'
-lPrecatWhisker {𝒞 = 𝒞} f _ _ p = cong (𝒞 .seq f) p
+lPrecatWhisker : {C : Precategory ℓ ℓ'} {x y z : C .ob} (f : C [ x , y ]) (g g' : C [ y , z ]) (p : g ≡ g') → f ⋆⟨ C ⟩ g ≡ f ⋆⟨ C ⟩ g'
+lPrecatWhisker {C = C} f _ _ p = cong (_⋆_ C f) p
 
 -- extend a cone on c by a morphism c'→c using precomposition
-coneMap : {𝒞 : Precategory ℓ ℓ'} {cspn : Cospan 𝒞} {c c' : ob 𝒞} (cn : Cone cspn c) (f : hom 𝒞 c' c) → Cone cspn c'
-coneMap {𝒞 = 𝒞} {cospan _ _ _ s₁ s₂} (cone p₁ p₂ sq) f =
-  cone (𝒞 .seq f p₁)  (𝒞 .seq f p₂) ((𝒞 .seq-α f p₁ s₁) ∙∙ lPrecatWhisker {𝒞 = 𝒞} f (𝒞 .seq p₁ s₁) (𝒞 .seq p₂ s₂) sq ∙∙ sym (𝒞 .seq-α f p₂ s₂))
+coneMap : {C : Precategory ℓ ℓ'} {cspn : Cospan C} {c c' : ob C} (cn : Cone cspn c) (f : C [ c' , c ]) → Cone cspn c'
+coneMap {C = C} {cospan _ _ _ s₁ s₂} (cone p₁ p₂ sq) f =
+  cone (f ⋆⟨ C ⟩ p₁)  (f ⋆⟨ C ⟩ p₂) ((C .seq-α f p₁ s₁) ∙∙ lPrecatWhisker {C = C} f (p₁ ⋆⟨ C ⟩ s₁) (p₂ ⋆⟨ C ⟩ s₂) sq ∙∙ sym (C .seq-α f p₂ s₂))
```

Comment:
Maybe rewrite using eq reasoning to increase readability?

Reply:
Not sure I completely understand the code for coneMap yet, might have to push this to a later PR